test/e2e/slider.cy.js

```diff
@@ -0,0 +1,24 @@
+describe('Slider', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/');
+  });
+
+  it('should display the slider component', function () {
+    const sliderComponents = this.data.components.filter((component) => component._component === 'slider')
+    sliderComponents.forEach((sliderComponent) => {
+      cy.visit(`/#/preview/${sliderComponent._id}`);
+      const bodyWithoutHtml = cy.helpers.stripHtml(sliderComponent.body)
+      cy.testContainsOrNotExists('.slider__body', bodyWithoutHtml)
+      
+      cy.testContainsOrNotExists('.slider__title', sliderComponent.displayTitle)
+      cy.testContainsOrNotExists('.slider__instruction', sliderComponent.instruction)
```

Comment:
```suggestion
    const sliderComponents = this.data.components.filter(component => component._component === 'slider')
    sliderComponents.forEach(sliderComponent => {
      cy.visit(`/#/preview/${sliderComponent._id}`);
      const stripHtml= cy.helpers.stripHtmlstripHtml;
      cy.testContainsOrNotExists('.slider__body', stripHtml(bodyWithoutHtml));
      cy.testContainsOrNotExists('.slider__title', stripHtml(sliderComponent.displayTitle));
      cy.testContainsOrNotExists('.slider__instruction', stripHtml(sliderComponent.instruction));
```

Please use the line ending `;` appropriately.